test/moonwall.config.json

```diff
@@ -860,17 +860,17 @@
         {
           "name": "polkadot",
           "type": "polkadotJs",
-          "endpoints": ["ws://127.0.0.1:8002"]
```

Comment:
FTR, did you bump into any issues that required switching from 127.0.0.1 to localhost?

Reply:
Yes, it is related to this change in chopsticks: https://github.com/AcalaNetwork/chopsticks/pull/826